release_notes/index.adoc

```diff
@@ -20,3 +20,11 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+{product-title} does not have non-backwards compatible changes and, therefore,
+does not drop features on versions earlier than v3. Some upgrades may require
+changes by the administrator. Clients will continue to work against major
+version servers. For example, a 3.1 client will work against 3.0, 3.1, and 3.2.
+Not all features are available in 3.1 clients against 3.2 servers.
```

Comment:
So what your looking for is a compatibility matrix:

X.____ | X.Y | X.Y+N | CLIENTS
X.Y__  |  [0]  | [2]
X.Y+N |  [1]  | [0]
Server

[0] Compatible
[1] Client may not be able to access server features
[2] Client may provide options and/features that may not be compatible with
the server being accessed.

On Aug 8, 2016 6:27 PM, "Aleksandar Kostadinov" <notifications@github.com>
wrote:

> In release_notes/index.adoc
> <https://github.com/openshift/openshift-docs/pull/2496#discussion_r73967631>
> :
>
> > @@ -20,3 +20,11 @@ technical notes, see the
> >  https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
> >  Enterprise 3.1 Release Notes].
> >  endif::[]
> > +
> > +[[release-versioning-policy]]
> > +== Versioning Policy
> > +{product-title} does not have non-backwards compatible changes and, therefore,
> > +does not drop features on versions earlier than v3. Some upgrades may require
> > +changes by the administrator. Clients will continue to work against major
> > +version servers. For example, a 3.1 client will work against 3.0, 3.1, and 3.2.
> > +Not all features are available in 3.1 clients against 3.2 servers.
>
> text only say one part "a 3.2 server may have additional capabilities that
> a 3.1 client cannot use.", the other part missing: "a 3.2 client may have
> additional capabilities that are not supported by a 3.1 server."
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2496/files/96bec963399fd1698069a0948690bca7e9c23aea#r73967631>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABUZmsF-cxpCKPSh_hRsWooTpsvlNaMOks5qd61egaJpZM4JLgeb>
> .
>